src/main/kotlin/no/nav/familie/ef/sak/behandling/henlegg/HenleggBehandlingController.kt

```diff
@@ -0,0 +1,123 @@
+package no.nav.familie.ef.sak.behandling.henlegg
+import no.nav.familie.ef.sak.AuditLoggerEvent
+import no.nav.familie.ef.sak.behandling.BehandlingService
+import no.nav.familie.ef.sak.behandling.dto.BehandlingDto
+import no.nav.familie.ef.sak.behandling.dto.HenlagtDto
+import no.nav.familie.ef.sak.behandling.dto.HenlagtÅrsak
+import no.nav.familie.ef.sak.behandling.dto.tilDto
+import no.nav.familie.ef.sak.fagsak.FagsakService
+import no.nav.familie.ef.sak.fagsak.domain.Fagsak
+import no.nav.familie.ef.sak.felles.util.isEqualOrAfter
+import no.nav.familie.ef.sak.infrastruktur.exception.feilHvis
+import no.nav.familie.ef.sak.infrastruktur.featuretoggle.FeatureToggleService
+import no.nav.familie.ef.sak.infrastruktur.featuretoggle.Toggle
+import no.nav.familie.ef.sak.infrastruktur.sikkerhet.SikkerhetContext
+import no.nav.familie.ef.sak.infrastruktur.sikkerhet.TilgangService
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.PersonopplysningerService
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.familie.kontrakter.felles.ef.StønadType
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.internal.TaskService
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDate
+import java.util.UUID
+
+@RestController
+@RequestMapping(path = ["/api/behandling"])
+@ProtectedWithClaims(issuer = "azuread")
+class HenleggBehandlingController(
+    private val behandlingService: BehandlingService,
+    private val fagsakService: FagsakService,
+    private val henleggService: HenleggService,
+    private val tilgangService: TilgangService,
+    private val featureToggleService: FeatureToggleService,
+    private val taskService: TaskService,
+    private val personopplysningerService: PersonopplysningerService,
+) {
+    @GetMapping("/{behandlingId}/henlegg/brev")
```

Comment:
Denne brukes utelukkende til forhåndsvisning, eller? Isåfall fint om både metodenavn og url gjenspeiler dette..?